ipyparallel/controller/hub.py

```diff
@@ -862,14 +873,13 @@ def save_iopub_message(self, topics, msg):
             name = content['name']
             s = '' if rec is None else rec[name]
             d[name] = s + content['text']
-
         elif msg_type == 'error':
             d['error'] = content
         elif msg_type == 'execute_input':
             d['execute_input'] = content['code']
         elif msg_type in ('display_data', 'execute_result'):
             d[msg_type] = content
-        elif msg_type == 'status':
+        elif msg_type in ('status', 'shutdown_reply', ):
```

Comment:
I think having which messages are logged here makes sense, but I can make an 'accepted list' in the `unregister_engine` if you want (with `('stream', 'error', 'execute_input', 'display_data', 'execute_result')`) or a not logged list (`('status', 'shutdown_reply', )`).  

I lean towards the latter.